src/test/java/com/example/enterpriseapplication/EnterpriseApplicationTests.java

```diff
@@ -13,7 +13,7 @@
 class EnterpriseApplicationTests {
 
 	@Autowired
-	private iRecipeService recipeService;
+	private IRecipeService recipeService;
```

Comment:
Changed name of service to fit naming convention set by videos and reference repo from class.

https://github.com/discospiff/SpringBootMicroservicesWithIntelliJIDEA/tree/master/src/main/java/com/myplantdiary/enterprise